_config.yml

```diff
@@ -217,6 +217,7 @@ kramdown:
 sass:
   sass_dir: _sass
   style: compressed # http://sass-lang.com/documentation/file.SASS_REFERENCE.html#output_style
```

Comment:
Gosh does this automagically minify the css?!

Reply:
as much as css can be minified! (it doesn't eg. simplify rules afaik, and you can't do the javascript thing of replacing all names with single characters, yes it does remove as much as it can https://sass-lang.com/documentation/cli/dart-sass/#style )